Gopkg.toml

```diff
@@ -64,7 +64,7 @@ required = [
 
 [[constraint]]
   name = "github.com/openshift/elasticsearch-operator"
-  branch = "master"
+  branch = "feature-es6x"
```

Comment:
This one needs to be reverted back to `master` and refresh with `dep ensure --update github.com/openshift/elasticsearch-operator` since es6 to master is merged in elasticsearch operator.

Reply:
in the interest of fewer file changes so things don't slip in we'll do this in a separate PR